src/components/CookieBanner/CookieBanner.css

```diff
@@ -0,0 +1,61 @@
+.util-cookie-banner-container {
+    flex-direction: column;
+    border-width: 1px solid;
+    border-color: red;
+    border-style: solid;
+    background-color: #FFFFFF;
+    position: fixed;
+    max-width: 480px;
+    min-width: 480px;
+    bottom: 0;
+    left: 0;
+    height: 100%;
+    box-shadow: 0px 2px 10px -3px #999;
+    font-size: 16px;
+    z-index: 1001;
+}
+
+.util-cookie-banner-container--show {
+    display: flex;
+    transition: transform 0.4s;
+    transform: translate(0%);
+}
+
+.util-cookie-banner-container--hide {
+    display: none;
+}
+
+.util-cookie-banner-container-hdr {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+.util-cookie-banner__cookie-icon {
+    position: fixed;
+    z-index: 2147483648;
```

Comment:
:). agree with the team on layer spaces. for example 0-1000(below modal), 1000 - 2000(above modal), 2000 - 3000 above all